Tasks/ServiceFabricDeploy/ServiceFabricSDK/Publish-UpgradedServiceFabricApplication.ps1

```diff
@@ -345,6 +345,8 @@ function Publish-UpgradedServiceFabricApplication
             }
         }
 
+        $upgradeStatus = Get-ServiceFabricApplicationUpgrade -ApplicationName $ApplicationName
```

Comment:
Consider:
$upgradeStatus = & $upgradeStatusFetcher

Seeing as we already have a lambda, may as well use it for consistency.